providers/alibaba/src/airflow/providers/alibaba/cloud/hooks/oss.py

```diff
@@ -93,7 +97,7 @@ def __init__(self, region: str | None = None, oss_conn_id="oss_default", *args,
 
     def get_conn(self) -> Connection:
         """Return connection for the hook."""
-        return self.oss_conn
+        return self.oss_conn  # type: ignore[return-value]
```

Comment:
This can/should be fixed -- if you change L37 to be Connection from SDK

Reply:
Handled it in cdf68858217b85aeb1b51df304dbdd73e0e3b7a3